src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1810,6 +1827,26 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
         return $bSaveSuccess;
     }
 
+    private function filterUnchangedFieldsFromPostData(array $postFieldsLanguageConvertedWithAccessRights, string $tableName, string $id): array
+    {
+        $escapedTableName = $this->databaseConnection->quoteIdentifier($tableName);
+        $dataBeforeUpdateQuery = 'SELECT * FROM '.$escapedTableName.' WHERE `id` = :id';
```

Comment:
Isn't this data already available in one of the instance fields?

And/or could be preserved befor the update/write is started?

(Having a SELECT query before every UPDATE sounds at least doubled - performance-wise.)